### PR TITLE
Added blog post about Thanos Mentoring Office Hours.

### PR DESCRIPTION
**Pull request by bwplotka**
Signed-off-by: Bartlomiej Plotka <bwplotka@gmail.com>

### CODE REVIEW

File: web/content/posts/thanos-mentoring-office-hours.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+featuredImage: "/images/blog/thanos-mentoring-office-hours/uche-overview.png"
+---
+
+During Summer of 2020, on top of our full-time work, on-call rotations, open source project's work, preparing conference talks and ☕ we had extra
+duties: `75%` the [Thanos](https://thanos.io) Team ([Giedrius](https://giedrius.blog/), [Lucas](https://github.com/squat), [Kemal](https://kakkoyun.me/), 
+[Matthias](https://matthiasloibl.com/), [Povilas](https://povilasv.me/) and me) and a few members of [Prometheus](https://prometheus.io) Team ([Callum](https://github.com/cstyan),
+[Chris](https://github.com/csmarchbanks) and me) were mentoring amazing people in their first steps in the Open Source! 🎉
+We taught mostly students but also already experienced developers, all new to the [CNCF](https://www.cncf.io/) space and projects. 
+
+We very are grateful that this year's edition was full of smart, diverse and incredibly curious candidates that applied for Thanos and Prometheus
```
Comment (thisisobate) [inline web/content/posts/thanos-mentoring-office-hours.md:22]: small nit: I think it should be "we are very grateful" instead of "we very are grateful"

Comment (yashrsharma44) [inline web/content/posts/thanos-mentoring-office-hours.md:22]: _are very_

File: web/content/posts/thanos-mentoring-office-hours.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+mentoring through [Google Summer of Code (GSoC)](https://developers.google.com/open-source/gsoc) and [Community Bridge](https://communitybridge.org/) programs.
+ 
+A longer blog post about learnings and tips that might be useful for other projects which willing to participate in such mentoring/internships is in progress, so stay tuned!
+ 
+However, today we present you the first actionable result of our learnings: **Weekly Thanos Mentoring Office Hours**.
+ 
+## Why?
+
+Long story, short, as part of the mentoring we scheduled a weekly meeting with each of our student for the duration of the program.
```
Comment (yashrsharma44) [inline web/content/posts/thanos-mentoring-office-hours.md:31]: _Long story short, _

File: web/content/posts/thanos-mentoring-office-hours.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+All engineers, new to the CNCF or open source, will be able to apply from September 21 for another Community Bridge Edition ([details](https://github.com/cncf/mentoring/blob/master/communitybridge/2020/q3-q4/README.md))
+
+We also plan to participate in the program called *Beyond* which allows students to be mentored by [Red Hat](http://redhat.com/) engineers on 
+exciting open source topics! ([details](https://research.redhat.com/blog/2020/05/24/open-source-development-course-and-devops-methodology/))
+
+## Summary
+
+As mentioned before, this is just a quick announcement. Ask us in few weeks how it goes, but we, Thanos maintainers are
+definitely super exciting for offering Mentoring Office Hours. 🤩
```
Comment (yashrsharma44) [inline web/content/posts/thanos-mentoring-office-hours.md:91]: Last symbol not visible

**Review by thisisobate — COMMENTED** [review]
small nit! ;)

File: web/content/posts/thanos-mentoring-office-hours.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+- posts
+title: Thanos Announces Weekly Mentoring Office Hours
+weight: 1
+categories:
+- open-source
+featuredImage: "/images/blog/thanos-mentoring-office-hours/uche-overview.png"
+---
+
+During Summer of 2020, on top of our full-time work, on-call rotations, open source project's work, preparing conference talks and ☕ we had extra
+duties: `75%` the [Thanos](https://thanos.io) Team ([Giedrius](https://giedrius.blog/), [Lucas](https://github.com/squat), [Kemal](https://kakkoyun.me/),
```
Comment (thisisobate) [inline web/content/posts/thanos-mentoring-office-hours.md:17]: 75% of the...

File: web/content/posts/thanos-mentoring-office-hours.md [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+weight: 1
+categories:
+- open-source
+featuredImage: "/images/blog/thanos-mentoring-office-hours/uche-overview.png"
+---
+
+During Summer of 2020, on top of our full-time work, on-call rotations, open source project's work, preparing conference talks and ☕ we had extra
+duties: `75%` the [Thanos](https://thanos.io) Team ([Giedrius](https://giedrius.blog/), [Lucas](https://github.com/squat), [Kemal](https://kakkoyun.me/), 
+[Matthias](https://matthiasloibl.com/), [Povilas](https://povilasv.me/) and me) and a few members of [Prometheus](https://prometheus.io) Team ([Callum](https://github.com/cstyan),
+[Chris](https://github.com/csmarchbanks) and me) were mentoring amazing people in their first steps in the Open Source! 🎉
```
Comment (thisisobate) [inline web/content/posts/thanos-mentoring-office-hours.md:19]: ...first steps into Open Source.

File: web/content/posts/thanos-mentoring-office-hours.md [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+We taught mostly students but also already experienced developers, all new to the [CNCF](https://www.cncf.io/) space and projects. 
+
+We very are grateful that this year's edition was full of smart, diverse and incredibly curious candidates that applied for Thanos and Prometheus
+mentoring through [Google Summer of Code (GSoC)](https://developers.google.com/open-source/gsoc) and [Community Bridge](https://communitybridge.org/) programs.
+ 
+A longer blog post about learnings and tips that might be useful for other projects which willing to participate in such mentoring/internships is in progress, so stay tuned!
```
Comment (thisisobate) [inline web/content/posts/thanos-mentoring-office-hours.md:25]: ...other projects which are willing

File: web/content/posts/thanos-mentoring-office-hours.md (hunk 2 above)
Comment (thisisobate) [inline web/content/posts/thanos-mentoring-office-hours.md]: ..as part of the mentoring, we...

File: web/content/posts/thanos-mentoring-office-hours.md [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+
+![My casual week during this summer. Red boxes: mentoring 1:1s.](/images/blog/thanos-mentoring-office-hours/calendar-mentoring.png)
+
+Despite extra time spent on mentoring, I personally got used to those extra meetings pretty quickly and had an amazing experience running them with co-mentors.
+
+> The main reason why mentoring and meetings were so satisfying for me is that we could see the mentees growing every week in terms of both technical and soft skills.
+> Every meeting was fully utilized thanks to energetic and passionated students and we could see that mentees literally respect and fully use our time. 🤗
+
+In fact, it was so addictive and fun that when GSoC and Community Bridge programs finished, it was just super awkward to... just stop our mentoring like that. 😱
+Most of our students had still some PRs open and the next tasks were still within the reach and quite exciting for all us. `Project is never ready. You can always improve things!`.
```
Comment (thisisobate) [inline web/content/posts/thanos-mentoring-office-hours.md:41]: ...all of us

**Review by thisisobate — COMMENTED** [review]
small nits!
Overall, this is a great article indeed! 🎉💯

**Comment by bwplotka** [comment]
Thanks All for reviews :hugs:

**Pull request merged**